src/main/resources/templates/ltft/admin.html

```diff
@@ -3,151 +3,180 @@
+              <dd class="nhsuk-summary-list__value" th:text="${pd?.skilledWorkerVisaHolder} ?: _">Not Given</dd>
+            </div>
+          </dl>
+        </div>
+      </div>
+      <div class="nhsuk-card">
+        <div class="nhsuk-card__content">
+          <h2 class="nhsuk-card__heading">Programme Details</h2>
+          <dl class="nhsuk-summary-list" th:with="programme=${var.programmeMembership}">
+            <div class="nhsuk-summary-list__row">
+              <dt class="nhsuk-summary-list__key">Programme Name</dt>
+              <dd class="nhsuk-summary-list__value" th:text="${programme?.name} ?: _">Not Given</dd>
+            </div>
+            <div class="nhsuk-summary-list__row">
+              <dt class="nhsuk-summary-list__key">Start Date</dt>
+              <dd class="nhsuk-summary-list__value" th:text="${#temporals.format(programme?.startDate, 'dd MMMM yyyy')} ?: _">Not Given</dd>
+            </div>
+            <div class="nhsuk-summary-list__row">
+              <dt class="nhsuk-summary-list__key">End Date</dt>
+              <dd class="nhsuk-summary-list__value" th:text="${#temporals.format(programme?.endDate, 'dd MMMM yyyy')} ?: _">Not Given</dd>
+            </div>
+            <div class="nhsuk-summary-list__row">
+              <dt class="nhsuk-summary-list__key">WTE (Current)</dt>
+              <dd class="nhsuk-summary-list__value" th:text="${programme?.wte} ?: _">Not Given</dd>
+            </div>
+          </dl>
+        </div>
+      </div>
+      <div class="nhsuk-card">
+        <div class="nhsuk-card__content">
+          <h2 class="nhsuk-card__heading">Proposed Changes</h2>
+          <dl class="nhsuk-summary-list" th:with="change=${var.change}">
+            <div class="nhsuk-summary-list__row">
+              <dt class="nhsuk-summary-list__key">WTE</dt>
+              <dd class="nhsuk-summary-list__value" th:text="${change?.wte} ?: _">Not Given</dd>
+            </div>
+            <div class="nhsuk-summary-list__row">
+              <dt class="nhsuk-summary-list__key">Start Date</dt>
+              <dd class="nhsuk-summary-list__value" th:text="${#temporals.format(change?.startDate, 'dd MMMM yyyy')} ?: _">Not Given</dd>
+            </div>
+            <div class="nhsuk-summary-list__row">
+              <dt class="nhsuk-summary-list__key">End Date</dt>
+              <dd class="nhsuk-summary-list__value" th:text="${#temporals.format(change?.endDate, 'dd MMMM yyyy')} ?: _">Not Given</dd>
+            </div>
+            <div class="nhsuk-summary-list__row">
+              <dt class="nhsuk-summary-list__key">Programme end date</dt>
+              <dd class="nhsuk-summary-list__value" th:text="${#temporals.format(change?.cctDate, 'dd MMMM yyyy')} ?: _">Not Given</dd>
+            </div>
+          </dl>
+        </div>
+      </div>
+      <div class="nhsuk-card">
+        <div class="nhsuk-card__content">
+          <h2 class="nhsuk-card__heading">Reasons</h2>
```

Comment:
Is it the th:block stuff which is throwing the column widths out? Its not a big deal at this point, but it looks odd that the list key entries are so much wider than they need to be in the Discussions and Reasons sections

Reply:
Possibly 🤷‍♂️
`th:block` elements disappear, but i think there is some weirdness with the whitespace left over from formatting them.

I'll check what happens if I one-liner it all 😅